File: C-Programming/Module_02/even_odd.c
#include <stdio.h>
int main()
{
    int a; printf("Enter a number: "); scanf("%d",&a) ;
    if(a%2==0)
    {
        printf("This is an Even number.\n") ;
    }else{
        printf("It is a Odd number.\n") ;
    }

    return 0;
}

File: Java/Book/even_odd.java
import java.util.Scanner;
public class even_odd {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();
        if(n%2==0){
            System.out.println("Even Number "+n);
        }else{
            System.out.println("Odd Number "+n);
        }
    }

}
